src/main/kotlin/three/consulting/epoc/service/CustomerService.kt

```diff
@@ -16,4 +16,17 @@ class CustomerService(private val customerRepository: CustomerRepository) {
         }
         return null
     }
+
+    fun createCustomer(customerData: CustomerDTO): CustomerDTO? {
```

Comment:
No need for the nullable return value. You either return the saved customer or throw an exception. Also you could use a more descriptive variable name. `customerData` doesn't really say if it's entity or a dto. Maybe the ones coming from controller could be something like `customerRequest`/`customerResponse` etc depending on the case?